apps/financroo-tpp/payments.go

```diff
@@ -95,10 +101,34 @@ func (o *OBBRClient) CreatePayment(c *gin.Context, data interface{}, accessToken
 		return PaymentCreated{}, errors.Wrapf(err, "failed to call pix payments endpoint")
 	}
 
+	if payload, ok = paymentCreatedResponse.Payload.(map[string]interface{}); !ok {
```

Comment:
I do not like this but with new client for OBBR response payload become interface{}